components/provisioner/cmd/main.go

```diff
@@ -177,6 +181,26 @@ func main() {
 	dbsFactory := dbsession.NewFactory(connection)
 	installationService := installation.NewInstallationService(cfg.ProvisioningTimeout.Installation, installationHandlerConstructor, cfg.Gardener.ClusterCleanupResourceSelector)
 
+	downloadManager := download.NewManager(&sync.Mutex{}, download.Config{
+		KymaURL:              "https://github.com/kyma-project/kyma",
+		ResourcesPathTmp:     "/app/downloads/%s",
```

Comment:
It doesn't work as there is not volume with write permissions mounted to the pod. Provisioner's deployment needs to be modified.